contrib/linux/sensors/file_watch_sensor.yaml

```diff
@@ -16,6 +16,11 @@
             description: "Path to the file to monitor"
             type: "string"
             required: true
+          follow:
```

Comment:
I wonder if we should use some other parameter name since in tail context, --follow / -f usually implies a slightly different thing.

Reply:
Maybe `sticky`?

Reply:
The parameter name `sticky` doesn't really make a whole lot of sense - it's closing the current (now non-existent) file, opening the new file, seeking to the same place, and continuing to tail from there. I think some variation on "follow" is a better description of the actual semantics.

How about `follow_on_move`?